src/App.css

```diff
@@ -36,4 +35,4 @@
   to {
     transform: rotate(360deg);
   }
-}
+} */
```

Comment:
- [ ] Lets not submit commented out code which makes our codebase untidy. Kindly fix this too